nmigen/hdl/cd.py

```diff
@@ -21,6 +21,8 @@ class ClockDomain:
         If ``True``, the domain does not use a reset signal. Registers within this domain are
         still all initialized to their reset state once, e.g. through Verilog `"initial"`
         statements.
+    clock_edge : str
```

Comment:
The actual parameter is called `clk_edge`.

Reply:
(But maybe it should be called `clock_edge` instead? We have a `rst` signal and `reset_less` parameter.)

Reply:
derp. i fix. that's what i get for adding extras :p

Reset polarity could be `rst_edge` for consistency... but it's not really an edge-active signal, so `rst_pol`?

Reply:
I was thinking `rst_active="pos"`/`="neg"`, since resets can be both synchronous and asynchronous.

Reply:
Sure, that's reasonable. I don't have too strong of opinions on this subject.